sys/src/cmd/mv.c

```diff
@@ -53,6 +53,14 @@ main(int argc, char *argv[])
 		exits("bad usage");
 	}
 
+	/* Skip -f */
+	if(argv[1][0] == '-' && argv[1][1] == 'f' && argv[1][2] == 0) {
```

Comment:
I do think I like it better. Another option would be to have some compat/bin directory that gets bind -b'ed onto /bin, that could just remove the '-f' option and invoke the real /$cputype/bin/mv:

#!/bin/rc
if ( ~ $1 '-f' ) { shift }
exec /$cputype/bin/mv $*